README.md

```diff
@@ -89,10 +130,15 @@ To run it from your desktop (Windows, Mac, Linux) it can be a bit more involved.
 * Windows 10 SDK (10.0.16299.0)
 * MSVC v141 - VS 2017 C++ x64/x86 build tools
 * MSVC v140 - VS 2015 C++ build tools (v14.00)

```

Comment:
Can you please sort them alphabetically? Would make it easier to search for the required components.